msmtools/util/sorted_schur.py

```diff
@@ -23,111 +23,64 @@ def _initialize_matrix(M, P):
         M.createDense(list(np.shape(P)), array=P)
 
 
-def top_eigenvalues(P, m, z='LM', tol=1e-16):
-    r"""
-    Sort `m+1` (if ``m < n``) or `m` (if ``m == n``) dominant eigenvalues 
-    up and check (if ``m < n``), if clustering into `m` clusters would split 
-    a complex conjugated pair of eigenvalues.
-    
-    Parameters
-    ----------
-    P : ndarray (n,n)
-        Transition matrix (row-stochastic).
-        
-    m : int
-        Number of clusters to group into.
-        
-    z : string, (default='LM')
-        Specifies which portion of the spectrum is to be sought.
-        The subspace returned will be associated with this part of the spectrum.
-        Options are:
-        'LM': the m eigenvalues with the largest magnitude are sorted up.
-        'LR': the m eigenvalues with the largest real part are sorted up.
+def _check_conj_split(m, eigenvalues):
```

Comment:
I guess now we don't need `find_twoblocks` in `gpcca` anymore...

Reply:
I removed the function and import `_check_conj_split` from `util`, to have a single source of truth. This is more clean, I think.